recipes/weave/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "weave" %}
+{% set version = "0.15.0" %}
+{% set sha256 = "4b7ac15abe2d1c0901c33db79d4bdcad8f2261198da36a48707b0023bca72f83" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py3k]  # Python2-only package
+  script: python setup.py install --record record.txt
+
+requirements:
+  build:
+    - python <3
```

Comment:
Drop the `<3` after `python` here and in the run requirement. The `# [py3k]` selector is a conda-build level guarantee that this package cannot be created for python 3, making this `<3` redundant